kids-chatbot/src/components/EntertainmentPanel.tsx

```
@@ -0,0 +1,121 @@
+      <div className="flex-1 p-4 space-y-4">
+        {entertainmentOptions.map((option) => (
+          <button
+            key={option.type}
+            onClick={() => handleEntertainment(option.type)}
+            disabled={loading === option.type}
+            className="w-full bg-gradient-to-r from-blue-50 to-purple-50 hover:from-blue-100 hover:to-purple-100 disabled:from-gray-100 disabled:to-gray-100 border border-blue-200 rounded-xl p-4 text-right transition-all transform hover:scale-105 disabled:scale-100"
+          >
+            <div className="flex items-center">
+              <div className="text-2xl ml-3">{option.emoji}</div>
+              <div className="flex-1">
+                <h3 className="font-bold text-gray-800">{option.title}</h3>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)